velox/exec/tests/AssertQueryBuilderTest.cpp

```diff
@@ -125,4 +126,31 @@ TEST_F(AssertQueryBuilderTest, hiveSplits) {
       .split(buildScanId, makeHiveConnectorSplit(buildFile->path))
       .assertResults("SELECT 2");
 }
+
+TEST_F(AssertQueryBuilderTest, encodedResults) {
+  VectorFuzzer::Options opts;
+  opts.vectorSize = 1000;
+  opts.nullRatio = 0.1;
+
+  VectorFuzzer fuzzer(opts, pool_.get());
+
+  // Dict(Array).
+  auto input =
+      makeRowVector({fuzzer.fuzzDictionary(fuzzer.fuzzFlat(ARRAY(INTEGER())))});
```

Comment:
alignment?

Reply:
I think it might be a display issue. make format-fix is happy.